themes/godotengine/pages/governance.htm

```diff
@@ -0,0 +1,84 @@
+title = "Governance"
+url = "/governance"
+layout = "more"
+description = "Governance model"
+is_hidden = 0
+==
+{##}
+{% put title %} Governance {% endput %}
+
+<div class="container">
+
+  <h3>Legal status</h3>
+  <p>
+    Godot is legally a part of the
+    <a href="https://sfconservancy.org">Software Freedom Conservancy</a>. As a
+    not-for profit charity, the Software Freedom Conservancy receives donations
+    and, based on their mission statement, use them for the benefit of the
+    project.
+    <strong>As no company is behind Godot, it can't be sold or purchased.</strong>
+  </p>
+  <p>
+    Within the organization sits the Godot
+    <a href="/contact#plc">Project Leadership Comittee</a>. This committee
+    decides and authorizes the usage of project funds (donations and grants).
+  </p>
+  <p>
+    <a href="/license">Godot's license is MIT</a>. This means you can do pretty
+    much everything with it, as long as you don't misrepresent the origin of the
+    software. In practice, even with that condition in place, you can do
+    anything you want as if you wrote Godot yourself.
```

Comment:
Should we mention here the licenses of third-party libraries used by Godot? They are permissive too but not the same.